hknweb/academics/models/course_surveys.py

```diff
@@ -0,0 +1,29 @@
+from django.db import models
+
+from .base_models import AcademicEntity
+
+
+class Question(AcademicEntity):
+    pass
+
+
+class Survey(AcademicEntity):
+    # reference attributes
+    survey_icsr = models.ForeignKey('ICSR', on_delete=models.CASCADE, related_name='survey_icsr')

```

Comment:
See comment about redundant prefixing.

Reply:
See above comment in response to comment about redundant prefixing.